tests/integration/security/external_ca/main_test.go

```diff
@@ -74,33 +76,52 @@ func TestMain(m *testing.M) {
 	// Refer to https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/
 	framework.NewSuite(m).
 		Label(label.CustomSetup).
-		RequireMinVersion(18).
-		// https://github.com/istio/istio/issues/22161. 1.22 drops support for legacy-unknown signer
-		RequireMaxVersion(21).
+		RequireSingleCluster().
```

Comment:
nit: I would still keep the min version of 1.18 because K8s signers was only introduced in that release

Reply:
fixed